HotelManagerment/com.spring.sample/src/com/spring/sample/controller/LoginController.java

```diff
@@ -2,20 +2,56 @@
 
 import java.util.Locale;
 
+import javax.servlet.http.HttpSession;
+
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.servlet.config.annotation.EnableWebMvc;
 
+import com.spring.sample.entity.Permission;
+import com.spring.sample.entity.Users;
+import com.spring.sample.service.PermissionService;
+import com.spring.sample.service.UsersService;
+
 @Controller
-@EnableWebMvc
+@RequestMapping("/login-page")
```

Comment:
```suggestion
@RequestMapping("/login")
```
thêm chữ page làm chi

Reply:
nó bị đụng với lại cái security chị làm sẳn mà em đang không biết fix cái phân quyền đó sao nên tạo cho nó đừng link riêng cho dễ quản lý